api/job/mock_job/mock_scheduler.go

```diff
@@ -1,73 +0,0 @@
-// Automatically generated by MockGen. DO NOT EDIT!
```

Comment:
It looks like this file needs to be removed rather than cleared out. The current code state indicates that this file still exists, which causes an issue if i run `go test ./...`

```
$ go test ./...
can't load package: package github.com/quintilesims/layer0/api/job/mock_job: 
mock_job/mock_scheduler.go:1:1: expected 'package', found 'EOF'
```

Reply:
This is removed on the branch, not sure why it's not reflecting that for you locally

Reply:
fwiw I believe a cleared out file would have 1 line of green, this is a removed file